droidlet/dashboard/web/build/VoxelWorld/world.html

```diff
@@ -54,11 +54,22 @@
         }
       }
 
+      function flashBlocks(bbox) {
+        if (flashBlocks != null) {
+          myWorld.flashBlocks(bbox)
+        }
+      }
+
       // document.getElementById("text-container").innerHTML=event.data
```

Comment:
Will you remove this commented code, or is it left in on purpose?

Reply:
I only changed one line of code, in `AgentThinking.js`, for this PR; everything else is the result of `yarn build`.  Lines 57-62 and 68-72 here are part of the changes made in #971 but it looks like updated build files were not yet checked in after that merge.

As to your note about the commented code, maybe @snyxan can comment, no pun intended.  I suspect at some point there could be a fruitful effort to clean up the dashboard files in general.